ecs.tf

```diff
@@ -78,11 +78,19 @@ resource "aws_ecs_service" "prometheus" {
     container_name   = "${local.roles[count.index]}-${var.name}"
     container_port   = var.prom_port
   }
+
+  service_registries {
+    registry_arn   = aws_service_discovery_service.prometheus[count.index].arn
+    container_name = "${local.roles[count.index]}-${var.name}"
```

Comment:
I know it's nitpicking but I think I'd prefer the interpolation the other way around so that we end up with "monitoring-master" and "monitoring-slave"

Reply:
Resolved